src/index.d.ts

```diff
@@ -1487,6 +1573,20 @@ declare namespace admin.auth {
      */
     deleteUser(uid: string): Promise<void>;
 
+    /**
+     * Deletes the users specified by the given identifiers.
+     *
+     * Deletes multiple existing users. Deleting a non-existing user won't
+     * generate an error. (i.e. this method is idempotent.)
+     *
+     * @param uids The `uids` corresponding to the users to delete.
+     *
+     * @return A Promise that resolves to the total number of successful/failed
+     *     deletions, as well as the array of errors that correspond to the
+     *     failed deletions.
+     */
+    deleteUsers(uids: string[]): Promise<admin.auth.DeleteUsersResult>;
```

Comment:
I am a bit confused. Why is `forceDelete` not included? This was approved by the API council in https://docs.google.com/document/d/1mG0ilmh8H-C6DYXA9oOQUro2MP0on_9k7Qf4oCLRyBk/